service/version_bundles.go

```diff
@@ -68,11 +68,77 @@ func NewVersionBundles() []versionbundle.Bundle {
 				},
 			},
 			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   false,
+			Deprecated:   true,
 			Name:         "kvm-operator",
 			Time:         time.Date(2017, time.October, 26, 16, 38, 0, 0, time.UTC),
 			Version:      "0.1.0",
-			WIP:          true,
+			WIP:          false,
+		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "calico",
+					Description: "Calico version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "docker",
+					Description: "Docker version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "etcd",
+					Description: "Etcd version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "kubedns",
+					Description: "KubeDNS version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "kubernetes",
+					Description: "Kubernetes version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "nginx-ingress-controller",
+					Description: "Nginx-ingress-controller version updated.",
+					Kind:        "changed",
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "kubedns",
+					Version: "1.14.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.8.4",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.9.0",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   false,
+			Name:         "kvm-operator",
+			Time:         time.Date(2017, time.October, 26, 16, 38, 0, 0, time.UTC),
+			Version:      "0.2.0",
```

Comment:
We may want to define this as `1.0.0` here like we did for AWS.

Reply:
anything that needs to be done in the resource router like you did in aws-operator?

Reply:
Actually yes, just like in AWS. Only note that we still use the default resource router so you have to do a bit copy/paste.

Reply:
Can you please take care of that? I don't know all the consequences of the changes I am making.

Reply:
Yeah but then I do it next week. I am quite on my limit right now with the TPR migration preparations.

Reply:
thx buddy! try to get some rest. You've been moving fast for a while now

Reply:
Note that @jgsqware is already helping here because he needs that as well. See https://github.com/giantswarm/kvm-operator/pull/308.